src/test/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/AnnotatedIntervalUtilsUnitTest.java

```diff
@@ -0,0 +1,91 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion;
+
+import com.google.common.collect.ImmutableSortedMap;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.reference.ReferenceUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+public class AnnotatedIntervalUtilsUnitTest extends GATKBaseTest {
+    @Test(dataProvider = "mergeTests")
+    public void testMerge(List<AnnotatedInterval> test1, List<AnnotatedInterval> gt) {
+        List<AnnotatedInterval> mergeTestResults = AnnotatedIntervalUtils.mergeRegions(test1,
+                ReferenceUtils.loadFastaDictionary(new File(ReferenceUtils.getFastaDictionaryFileName(hg19MiniReference))),
+                "__", l -> {});
+        Assert.assertEquals(mergeTestResults, gt);
+    }
+
+    @DataProvider(name = "mergeTests")
+    public Object [][] createMergeTests() {
+
+        return new Object[][] {
+            {
+                Arrays.asList(
```

Comment:
You could avoid some code duplication here by moving the `Arrays.asList()` call into your test method.

Reply:
Apologies if I am missing something, but it seems like I would have to do the following, which does not reduce code duplication:

```
    @DataProvider(name = "mergeTests")
    public Object [][] createMergeTests() {

        return new Object[][] {
            {
                    new AnnotatedInterval[]{
                            new AnnotatedInterval(new SimpleInterval("1", 100, 200),
                                    ImmutableSortedMap.of("Foo", "bar", "Foo1", "bar1")),
                            new AnnotatedInterval(new SimpleInterval("1", 100, 200),
                                    ImmutableSortedMap.of("Foo", "bar", "Foo1", "bar1"))
                    },
// ..........
```

No action

Reply:
@pshapiro4broad Comment above...

Reply:
You can avoid that by calling `new Object[][][] { ... }` E.g.
```
    @Test(dataProvider = "simpleTests")
    public void testSimpleTagging(AnnotatedInterval[] tumorSegments, AnnotatedInterval[] normalSegments,
                                  AnnotatedInterval[] gt) {

        final List<AnnotatedInterval> testResult = SimpleGermlineTagger.tagTumorSegmentsWithGermlineActivity(Arrays.asList(tumorSegments),
                Arrays.asList(normalSegments), "call",
                ReferenceUtils.loadFastaDictionary(new File(ReferenceUtils.getFastaDictionaryFileName(REF))), TEST_GERMLINE_TAGGING_ANNOTATION, 10);

        Assert.assertEquals(testResult, Arrays.asList(gt));
    }

    @DataProvider(name = "simpleTests")
    public Object[][] createSimpleTests() {
        return new Object[][][]{{
                {
                        // Tumor segments are assumed to be mutable.
                        new AnnotatedInterval(new SimpleInterval("1", 100, 200), Maps.newTreeMap(ImmutableSortedMap.of("call", "+"))),
                        new AnnotatedInterval(new SimpleInterval("1", 201, 300), Maps.newTreeMap(ImmutableSortedMap.of("call", "0")))
                },
                {
                        new AnnotatedInterval(new SimpleInterval("1", 100, 200), ImmutableSortedMap.of("call", "+")),
                        new AnnotatedInterval(new SimpleInterval("1", 201, 500), ImmutableSortedMap.of("call", "0"))
                },
                {
                        new AnnotatedInterval(new SimpleInterval("1", 100, 200), ImmutableSortedMap.of("call", "+", TEST_GERMLINE_TAGGING_ANNOTATION, "+")),
                        new AnnotatedInterval(new SimpleInterval("1", 201, 300), ImmutableSortedMap.of("call", "0", TEST_GERMLINE_TAGGING_ANNOTATION, "0"))
                }
        },
       ...
        };
    }
```

Although now that I've written it out it doesn't really format that much shorter. I'll leave it up to you.

Reply:
Done

Reply:
(no action)